lab_1_classify_by_unigrams/main.py

```diff
@@ -105,6 +176,26 @@ def detect_language(
 
     In case of corrupt input arguments, None is returned
     """
+    if unknown_profile is not None and profile_1 is not None and profile_2 is not None:
```

Comment:
the same

Reply:
I have "return None" at the end of the cycle. I tried to place it in a different way, but got mistakes with returns (there was something like  "you cannot return value and None").

Reply:
.

Reply:
done